.gitignore

```diff
@@ -5,6 +5,7 @@ build/
 [Oo]bj/
 [Hh]elp/
 NuGetPackages/
+[pP]ackages
```

Comment:
This was removed, the issue is that newer versions of NuGet are ignoring the repositoryPath redirect.  This happens to occur if using VS 2015 (Nuget 3.3) with restore packages on.  My apologies for not seeing the packages having been committed at the root level of the repo.